crates/bevy_gltf/src/loader.rs

```diff
@@ -1194,35 +1229,37 @@ fn primitive_name(mesh: &gltf::Mesh, primitive: &Primitive) -> String {
 }
```

Comment:
I didn't change `primitive_name` to match the changes to the asset labels because I wanted to limit scope to only the asset labels.  This appears to be used when naming the entities within a Scene.  This can be changed to match if anyone has a preference here.